backend/crates/eiffelvis_gen/src/eiffel_vocabulary.rs

```diff
@@ -0,0 +1,42 @@
+use crate::event_set::{Event, EventSet, Link};
+
+use rand::Rng;
+
+pub struct EiffelVocabulary;
+
+const EVENT_TYPES: [&str; 23] = [
+    "ActC", "ActF", "ActS", "ActT", "AnnP", "ArtC", "ArtP", "ArtR", "TCC", "TCF", "TCS", "TCT",
```

Comment:
Think this change undid the event type renames

Reply:
Probably a consequence on working on this late in the evening :( Thanks for catching it!